packages/web/src/components/spa/AboutSection/AboutSection.jsx

```diff
@@ -3,7 +3,7 @@ import './about_section.css';
 
 export const AboutSection = () => {
     return (
-        <div className="about-container" id="AboutSection">
+        <div className="about-container" id="about">
             <div className="about-image">
```

Comment:
To what I noticed, the about section anchor in the landing page is mistakenly connected to the one from the hackaubg instead. Hence, when you click the About anchor in Hackaubg nothing happens, but when you got then to the landing page, it loads immediately to the about us section.